app/controllers/api/v2/override_values_controller.rb

```diff
@@ -11,6 +12,8 @@ class OverrideValuesController < V2::BaseController
       before_action :return_if_smart_mismatch, :only => [:index, :create, :show, :update, :destroy]
       before_action :return_if_override_mismatch, :only => [:show, :update, :destroy]
 
+      deprecate_param [:create, :update], [:override_value, :use_puppet_default], :skip_foreman
```

Comment:
We do want to use the new parameter in code but we are not changing the API, so we need to rename the parameter before it is processed by the controller.